components/_Progress-Bar/progress-bar.js

```diff
@@ -0,0 +1,63 @@
+const msdsProgressBar = {
+  //Get the base number for the progress bar
+  parentElement: '',
+  baseNumber: 0,
+  currentStep: 0,
+  startFromOne: false,
+  init: function(parentElementId, baseNumber, customProgressionStep) {
+    this.parentElement = document.getElementById(parentElementId)
+
+    if (customProgressionStep > 0) {
+      this.currentStep = customProgressionStep
+    } else if (this.startFromOne == true) {
+      this.currentStep = 1
+    } else {
+      this.currentStep = 0
+    }
+    this.baseNumber = baseNumber
+    this.updateProgressbarDisplay()
+  },
+  updateProgressbarDisplay: function() { //update the display of the prorgess bar (step and counter)
```

Comment:
I think the name of the method should be self-explanatory, the need for a comment like this is a code smell to me.